.github/workflows/check-commit-message.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
     if: github.event.pull_request.user.login != 'dependabot[bot]'
     steps:
     - name: Checkout code
-      uses: actions/checkout@1af3b93b6815bc44a9784bd300feb67ff0d1eeb3  # v6.0.0
```

Comment:
These changes to the build step versions seem like they may have been unintentional. If so, can you revert them?